src/components/StyledElements/MoreDots.js

```diff
@@ -103,6 +104,7 @@ export const DropDownMenu = ({ title, children }) => {
         aria-controls="long-menu"
         aria-haspopup="true"
         {...bindTrigger(popupState)}
+        {...restProps}
```

Comment:
I'm not sure whether or not we've defined a naming convention for this that it could be violating.  I'll leave that up to @gvan12 to decide

Reply:
{...restProps} is good with me. 👍